stdlib/LinearAlgebra/src/cholesky.jl

```diff
@@ -284,7 +284,7 @@ Stacktrace:
 [...]
 ```
 """
-function cholesky!(A::StridedMatrix, ::Val{false}=Val(false); check::Bool = true)
+function cholesky!(A:: Matrix, ::Val{false}=Val(false); check::Bool = true)
```

Comment:
DId you mean
```suggestion
function cholesky!(A::AbstractMatrix, ::Val{false}=Val(false); check::Bool = true)
```
here?